vita3k/renderer/src/gl/draw.cpp

```diff
@@ -185,6 +190,10 @@ void draw(GLState &renderer, GLContext &context, const FeatureState &features, S
         glBindFramebuffer(GL_FRAMEBUFFER, context.render_target->framebuffer[0]);
     }
 
+    if (both_side_fragment_program_disabled) {
+        sync_blending(context.record, mem);
```

Comment:
Hmm why need sync again?

Reply:
Restore color mask. I doubt there exist such crazy game but it can technically do setFragmentProgram -> disable side -> draw -> enable side -> draw.